frontend/device/generic/device.lua

```diff
@@ -228,11 +228,17 @@ function Device:onPowerEvent(ev)
         self.powerd:beforeSuspend()
         local UIManager = require("ui/uimanager")
         logger.dbg("Suspending...")
-        -- Mostly always suspend in portrait mode...
-        -- ... except when we just show an InfoMessage, it plays badly with landscape mode (c.f., #4098)
+        -- Mostly always suspend in Portrait/Inverted Portrait mode...
+        -- ... except when we just show an InfoMessage, it plays badly with Landscape mode (c.f., #4098)
         if G_reader_settings:readSetting("screensaver_type") ~= "message" then
             self.orig_rotation_mode = self.screen:getRotationMode()
-            self.screen:setRotationMode(0)
+            -- Leave Portrait & Inverted Portrait alone, that works just fine.
```

Comment:
Note to self: possibly could have gone with a getScreenMode == "portrait" here instead.

But I'm not a fan of getScreenMode anyway ;p (which basically only does a w > h check).